Corona-Warn-App/src/main/res/values-de/strings.xml

```diff
@@ -876,7 +876,7 @@
     <!-- XBUT: test result pending : remove the test button -->
     <string name="submission_test_result_pending_remove_test_button">"Test entfernen"</string>
     <!-- XHED: Page headline for negative test result next steps  -->
-    <string name="submission_test_result_negative_steps_negative_heading">"Ihr Testergebnis"</string>
+    <string name="submission_test_result_negative_steps_negative_heading">"Befund Negativ"</string>
```

Comment:
```suggestion
    <string name="submission_test_result_negative_steps_negative_heading">"Befund negativ"</string>
```
I can't see a reason for a capital "n" in this case, but I can't double check in the mockups. @janetback , can you please double check?